packages/web-components/fast-ssr/package.json

```diff
@@ -30,8 +30,8 @@
     "install-playwright-browsers": "npx playwright install"
   },
   "description": "A package for rendering FAST components outside the browser.",
-  "main": "./dist/esm/index.js",
-  "types": "./dist/dts/index.d.ts",
+  "main": "./dist/esm/exports.js",
+  "types": "./dist/dts/exports.d.ts",
   "private": true,
   "dependencies": {
     "@lit-labs/ssr": "^1.0.0-rc.2",
```

Comment:
One thing I was curious of is whether we needed this dependency. Seems like in the end there should only be some interfaces but no concrete code to the protocol for things to work together. Is that possible or will we continue to need this in the end?

Reply:
Right now the ElementRenderer extends an abstract class exported form lit-labs. I'm unsure if extending it directly is a requirement, but it does handle some element functionality for as.  I'm also using their DOM shim to try to align DOM API requirements.